src/component/Modal/ModalAdd.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import "./ModalAdd.css";
+import closeBtn from "../../images/close.svg";
+
+const ModalAdd = ({ folderNameData, closeModal }) => {
+  const [isClick, setIsClick] = useState(null);
+
+  const handleCloseModal = () => {
+    closeModal(false);
+  };
+
+  const handleClick = (id) => {
+    setIsClick(id);
+  };
+
+  return (
+    <>
+      <div className="modal-bg"></div>
```

Comment:
```suggestion
      <div className="modal-bg"/>
```
self closing 할 수 있겠네요 ㅎㅎ